js/src/data/utils.test.js

```diff
@@ -263,3 +264,34 @@ describe( 'mapReportFieldsToPerformance', () => {
 		} );
 	} );
 } );
+
+describe( 'getConnectedJetpackInfo', () => {
+	it( 'When jetpack in not connected, should return an empty string', () => {
+		expect( getConnectedJetpackInfo( { active: 'no' } ) ).toBe( '' );
+	} );
```

Comment:
💅 ❔ Just wondering whether we should use a `describe` block for this so that it is consistent with the other `describe` block.

```suggestion
	describe( 'For non-connected jetpack', () => {
		it( 'should return an empty string', () => {
			expect( getConnectedJetpackInfo( { active: 'no' } ) ).toBe( '' );
		} );
	} );
```

Reply:
I would say both works for me. And `describe` would be more meaningful to group the related test cases if they have a commonality in the purpose or premise. However, based on the consistency of the typography or code blocks, it is not a priority when writing tests, especially since one more layer group has only one case.

The same blocks structure:

https://github.com/woocommerce/google-listings-and-ads/blob/20247e60a491f2546bcaee5862d30f678b01557f/js/src/reports/utils.test.js#L18-L33